libs/blocks/section-metadata/section-metadata.js

```diff
@@ -51,11 +52,19 @@ async function handleStickySection(sticky, section) {
   }
 }
 
+function handleAutoUps(section) {
+  const nonColumns = ['fullwidth', 'full-width', 'section-metadata'];
+  const columns = [...section.children].filter(({ classList }) => !nonColumns.some((name) => classList.contains(name)));
```

Comment:
```suggestion
  const columns = [...section.children].filter(
    ({ classList }) => !nonColumns.some((name) => classList.contains(name)),
  );
```
I'm still seeing a max length eslint warn on my end. There are a few others in this file, not related to your PR, but maybe take a peek.